tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1024,18 +1025,17 @@ def runTest(self):
                 print(list(map(operator.sub, pg_cntrs, pg_cntrs_base)),
                       file=sys.stderr)
                 for i in range(0, PG_NUM):
+                    # In DNX, ip2me packets trapped to cpu mapped to TC 1 -> PG 0 and
```

Comment:
As discussed, lets try this logic -

i => 0 -> 7
pg => 0,3,4
dscps => 62,1,1

DNX/Chassis: 
   pg = 0 => Some extra packets with unmarked TC
   pg = 4 => Extra packets for LACP/BGP packets
   pg = 7 => packets from cpu to front panel ports

 if non chassis:
   if i == pg:
         assert(pg_cntrs[pg] == pg_cntrs_base[pg] + len(dscps))
   else:
          assert(pg_cntrs[i] == pg_cntrs_base[i])
 else
   i = {3}
      assert(pg_cntrs[pg] == pg_cntrs_base[pg] + len(dscps))
   i = {0,4,7}
      assert(pg_cntrs[pg] >= pg_cntrs_base[pg] + len(dscps))
   else {1,2,5,6}
      assert(pg_cntrs[i] == pg_cntrs_base[i])